packages/amplication-git-utils/src/git/git-client.ts

```diff
@@ -0,0 +1,36 @@
+import { simpleGit, SimpleGit } from "simple-git";
+
+export class GitClient {
```

Comment:
I would rename it to GitRepository.

Reply:
Do we actually need that class? Why do we wrap simple git? Do we expect that we may change a library for working with git?